CHANGELOG.md

```diff
@@ -6,454 +6,473 @@ All notable changes to this project will be documented in this file.
 
 ## Added
 
-- Automate
-  - Import Rake task `OVERWRITE` argument: Setting  `OVERWRITE=true` removes the target domain prior to import
-  - New `extend_retires_on` method: Used by Automate methods to set a retirement date to specified number of days from today, or from a future date.
-  - Service model updates
-    - MiqAeServiceHardware
-    - MiqAeServicePartition
-    - MiqAeServiceVolume
+### Automate
+
+- New methods
+  - `extend_retires_on` method: Used by Automate methods to set a retirement date to specified number of days from today, or from a future date.
+  - `taggable?` to programmatically determine if a Service Model class or instance is taggable.
+- Service model updates
+  - MiqAeServiceHardware
+  - MiqAeServicePartition
+  - MiqAeServiceVolume
```

Comment:
MiqAeServicePartition and MiqAeServiceVolume are new models.  (There is a "New Service Models" section below.)

MiqAeServiceHardware was updated to expose a relationship to partitions.